trade-finance-manager-ui/server/controllers/utilisation-reports/find-reports-by-year/index.test.ts

```diff
@@ -144,13 +144,41 @@ describe('controllers/utilisation-reports/find-reports-by-year', () => {
       });
     });
 
+    it("renders the 'find-utilisation-reports-by-year.njk' view with when there is a query in the original URL with validation errors", async () => {
+      // Arrange
+      const { res, req } = httpMocks.createMocks({
+        session: requestSession,
+        query: { bank: '', year: '' },
+        originalUrl: '/utilisation-reports/find-reports-by-year?',
+      });
+
+      const expectedBankError = 'Select a bank';
+      const expectedYearError = 'Enter a valid year';
+      const expectedErrorSummary = [
+        { text: expectedBankError, href: '#bank' },
+        { text: expectedYearError, href: '#year' },
+      ];
+
+      jest.mocked(api.getAllBanks).mockResolvedValue(BANKS);
+
+      // Act
+      await getFindReportsByYear(req, res);
+
+      // Assert
+      expect(res._getRenderView()).toEqual('utilisation-reports/find-utilisation-reports-by-year.njk');
+      expect((res._getRenderData() as FindUtilisationReportsByYearViewModel)?.errorSummary).toStrictEqual(expectedErrorSummary);
+      expect((res._getRenderData() as FindUtilisationReportsByYearViewModel)?.bankError).toBe(expectedBankError);
+      expect((res._getRenderData() as FindUtilisationReportsByYearViewModel)?.yearError).toBe(expectedYearError);
+    });
+
     it("renders the 'find-utilisation-reports-by-year.njk' view with required data when there are query params with errors", async () => {
       // Arrange
       const yearQuery = '20';
 
       const { res, req } = httpMocks.createMocks({
         session: requestSession,
         query: { year: yearQuery },
+        originalUrl: '/utilisation-reports/find-reports-by-year?',
```

Comment:
May be worth storing this in a variable instead of repeating it in each test setup?